src/modules/dashboard/components/ActionsPage/StakingWidget/StakingWidget.tsx

```diff
@@ -282,9 +288,15 @@ const StakingWidget = ({
                 text={MSG.stakeButton}
               />
               <Button
-                appearance={{ theme: 'danger', size: 'medium' }}
-                text={MSG.objectButton}
-                disabled={!canUserStake}
+                appearance={{
+                  theme: previousTotalStakeStep ? 'ghost' : 'danger',
```

Comment:
I think it's not `ghost` theme for the back button. But we can reuse the one that I have added in my PR since I assume we will be using my logic for this as well - there is just many more details to what i've added.

<img width="375" alt="Screenshot 2021-04-27 at 15 34 49" src="https://user-images.githubusercontent.com/34057551/116259930-2988bb80-a76e-11eb-83fa-25edc64e91a0.png">